README.md

```diff
@@ -174,7 +174,7 @@ The test tool has the following limitations. If the DUT falls under these limita
 | **Requirement** | **T2IAPI Manipulation**                                                              | 
 |-----------------|--------------------------------------------------------------------------------------|
 | R0025_0         | SetComponentActivation                                                               |
-| R0029           | SetAlertConditionPresence                                                            |
+| R0029           | SetAlertActivation, SetAlertConditionPresence                                        |
```

Comment:
Sollte das Requirement nicht R0029_0 heissen?

Reply:
indeed, changed it